configs/crowbar.conf

```diff
@@ -9,5 +9,10 @@
     ErrorDocument 422 /422.html
     ErrorDocument 500 /500.html
 
+    IncludeOptional /etc/apache2/conf.d/crowbar-ui.conf.partia[l]
```

Comment:
Otherwise the path is not allowed to be empty,  IncludeOptional is unfortunately not enough in this case. See https://github.com/crowbar/crowbar-core/pull/663